src/extension.ts

```diff
@@ -2,6 +2,11 @@
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
 import * as k8s from 'vscode-kubernetes-tools-api';
+import * as path from 'path';
+
+// home directory
+const HOME = require('os').homedir();
```

Comment:
you can (and should!) use the import syntax instead of the old require syntax:

ie:

import { homedir } from 'os';